resources/eventing/values.yaml

```diff
@@ -14,6 +14,11 @@ global:
   # if true BEB is enabled else NATS is enabled
   isBEBEnabled: false
 
+  # eventTypePrefix is the prefix of the eventType
+  # note that the eventType format is: eventTypePrefix.applicationName.eventName.eventVersion
+  # for example: sap.kyma.custom.myapp.order.created.v1 (where the eventName is order.created)
+  eventTypePrefix: "sap.kyma.custom"
```

Comment:
For the open source offering, an empty "eventTypePrefix" is preferable as the current default one: "sap.kyma.custom". Of course this should be configurable as a global value and used by the calling applications, if necessary

Reply:
A following issue was created: https://github.com/kyma-project/kyma/issues/10579